### PR TITLE
Translation to quiz in hindi

### PR DESCRIPTION
**Pull request by innovation-platform**

### CODE REVIEW

**Comment by innovation-platform** [comment]
@Anurag-0-1-A  as I was facing some issues with my previous PR I have made a new PR and made all the suggested changes. Thank you.

**Comment by Anurag-0-1-A** [comment]
Screenshot_2021-07-21-12-13-09

It's showing 0 lines

**Comment by innovation-platform** [comment]
> Screenshot_2021-07-21-12-13-09
> 
> It's showing 0 lines @innovation-platform

Sorry for that there might be some problem I'll look into it.

File: quiz-app/src/assets/translations/hin.json [hunk 1]
```diff
@@ -0,0 +1,1017 @@
...
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान _______ हैं :",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:331]: API के समान _______ हैं

Why _______ ?

I think no need to underline

File: quiz-app/src/assets/translations/hin.json [hunk 2]
```diff
@@ -0,0 +1,1017 @@
...
+							{
+								"answerText": "एक उच्च शक्ति डिवाइस को नियंत्रित करने के लिए एक कम शक्ति डिवाइस",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "कम पावर डिवाइस को नियंत्रित करने के लिए एक उच्च शक्ति उपकरण",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:697]: I am not getting the meaning of this option

"उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",

File: quiz-app/src/assets/translations/hin.json [hunk 3]
```diff
@@ -0,0 +1,1017 @@
...
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT हब के नाम अद्वितीय होने चाहिए:",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:801]: अद्वितीय may be replaced with अनोखा for easy understanding

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]
@innovation-platform, Some minor changes required

File: quiz-app/src/assets/translations/hin.json [hunk 4]
```diff
@@ -0,0 +1,1017 @@
...
+						]
+					}
+				]
+			},
+			{
+				"id": 19,
+				"title": "पाठ 10 - अपने पौधा को सुरक्षित रखें: प्री-लेक्चर क्विज",
+				"quiz": [
+					{
+						"questionText": "IoT डिवाइस हमेशा सुरक्षित होते हैं",
```
[the remark this answers is not in the document]
Comment (innovation-platform) [reply quiz-app/src/assets/translations/hin.json:918]: Yeah but the question is repeated in the en.json also. Please suggest me how to proceed.

[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:918]: @jimbobbennett, The question "IoT devices are always secure? True: False" is repeated. 

Kindly check Line 78 and 918 at id=2 and id=19 respectively of https://github.com/innovation-platform/IoT-For-Beginners/blob/hindi_quiz/quiz-app/src/assets/translations/en.json

File: quiz-app/src/assets/translations/hin.json [hunk 5]
```diff
@@ -694,7 +694,7 @@
 								"isCorrect": "false"
 							},
 							{
-								"answerText": "उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",
+								"answerText": "धावकों को दौड़ मे उनके बीच  एक बैटन को पारित करते हुये दौड़ने को",
```
Comment (Anurag-0-1-A) [inline quiz-app/src/assets/translations/hin.json:697]: I am sorry but I am not getting the meaning of this option.

"धावकों को दौड़ मे उनके बीच एक बैटन को पारित करते हुये दौड़ने को"

Kindly share the English option in reply to this comment for understanding it better.

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]
Thank You @innovation-platform for making the suggested changes.

One Change is required. Kindly check it

File: quiz-app/src/assets/translations/hin.json (hunk 5 above)
Comment (innovation-platform) [reply quiz-app/src/assets/translations/hin.json:697]: Runners to run a race passing a baton between them

Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:697]: I am not really sure whether this option is anywhere related to the IoT.

I would like to see the question in the English JSON file for my better understanding.

Comment (innovation-platform) [reply quiz-app/src/assets/translations/hin.json:697]: This option is not related to IoT. I t gives another meaning of relay.

Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:697]: Okay, Then it's fine.

Sorry for my misunderstanding.

Comment (jimbobbennett) [reply quiz-app/src/assets/translations/hin.json:697]: @Anurag-0-1-A - is this now ready to merge?

Comment (Anurag-0-1-A) [reply quiz-app/src/assets/translations/hin.json:697]: Yes

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**